docs/mkdocs/docs/technical/contributing.md

```diff
@@ -26,80 +95,50 @@ See [pybind11 configuration](#pybind11-configuration).
 | Ubuntu | 20.04, 22.04 | build-essential g++-10 libpcre3-dev libsasl2-dev libsodium-dev libkrb5-dev libcurl4-openssl-dev python3-dev |
 | Centos | 7 | devtoolset-10-gcc-c++ openssl-devel cyrus-sasl-devel devtoolset-10-libatomic-devel libcurl-devel python3-devel |
 
-Windows
--------
+Setting up Windows
+------------------
 We recommend using Visual Studio 2022 (or later) to install the compiler (MSVC v142 or newer) and tools
 (Windows SDK, CMake, Python).
 
 The Python that comes with Visual Studio is sufficient for creating release builds, but for debug builds, you will have
 to separately download from [Python.org](https://www.python.org/downloads/windows/).
 
-Ways to build the code
-======================
-First steps
------------
-On Windows, you need to [configure `git` to allow symbolic links](https://stackoverflow.com/a/59761201).
+Running Python tests
+--------------------
 
-Recursively clone the repository:
+With `python` pointing to a Python interpeter with `ArcticDB` installed/on the `PYTHON_PATH`:
 
 ```bash
-git clone https://github.com/man-group/ArcticDB.git
-cd ArcticDB
-git submodule init && git submodule update
+python -m pip install arcticdb[Testing]
+python -m pytest python/tests
 ```
 
-Quick check
------------
-If you just want to verify your setup, try running from the command line:
+Running C++ tests
+-----------------
 
-```bash
-# Recursively checkout ArcticDB
-# Make sure the Python version you want is the first on the PATH
+Configure ArcticDB with TEST=on:
 
-python -m pip install -ve .
+```bash
+cmake -DPython_EXECUTABLE=<path to python> -DTEST=ON --preset linux-debug cpp
 ```
 
-This will build and install ArcticDB in the current working directory.
-
-Compiling step by step
-----------------------
-During day-to-day development, it's quicker to rebuild only the part that changed.
+Note that `<path to python>` must point to a Python that is compatible with [Development.Embed](https://cmake.org/cmake/help/latest/module/FindPython.html). This will probably be the result of installing `python3-devel` from your distributions dependency manager. 
```

Comment:
Missing apostrophe distributions

Reply:
...my grammer isn't good enough for this. I'm removing the word `distribution` 😆